docs/topics/native/native-objc-interop.md

```diff
@@ -32,27 +32,137 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a framewor
 * See [Build final native binaries](multiplatform-build-native-binaries.md#declare-binaries) to see how to declare binaries.
 * Check out the [Kotlin Multiplatform sample project](https://github.com/Kotlin/kmm-basic-sample) for an example.
 
-### Hiding Kotlin declarations
+If you don't want to export Kotlin declarations to Objective-C and Swift, you can use special annotations, [`@HiddenFromObjC`](#hiding-kotlin-declarations-from-objective-c-and-swift)
+and [`@ShouldRefineInSwift`](#use-refining-in-swift).
```

Comment:
@ShouldRefineInSwift is for another purpose and the declaration is exported but hidden. Don't you mean @HiddenFromObjC and internal?

Reply:
Thank you! I agree that this paragraph is confusing, it's better to remove it